imagetagger/imagetagger/users/templates/users/view_user.html

```diff
@@ -19,6 +19,11 @@ <h3>Teams:</h3>
                     </ul>
                 </div>
             </div>
+        {% if own_profile %}
+            <form method="get" action="/users/user/{{ user.id }}/delete/">
+                <button class="btn btn-danger" type="submit">Delete this account</button>
```

Comment:
I haven't worked with Django forms in a long time. Enlighten me: Why is this a form and not an anchor?

Reply:
Oops, that was a mistake. I fixed it.